lib/dialects/postgres/query-generator.js

```diff
@@ -877,6 +877,10 @@ var QueryGenerator = {
       return this.handleSequelizeMethod(value);
     }
 
+    if (field && field.type && value) {
+      field.type.validate(value);
```

Comment:
That would be very nice :) - There might be some places in abstract you need to change to actually pass the field to escape :)